windows/src/developer/TIKE/child/Keyman.Developer.UI.UfrmModelEditor.dfm

```diff
@@ -211,15 +247,67 @@ inherited frmModelEditor: TfrmModelEditor
               'custom')
           end
           object memoComments: TMemo
-            Left = 98
-            Top = 80
-            Width = 506
+            Left = 106
+            Top = 160
+            Width = 498
             Height = 87
             Anchors = [akLeft, akTop, akRight]
             BevelOuter = bvNone
-            TabOrder = 2
+            TabOrder = 6
             OnChange = memoCommentsChange
           end
+          object cbInsertAfterWord: TComboBox
+            Left = 106
+            Top = 80
+            Width = 145
+            Height = 21
+            TabOrder = 2
+            OnChange = cbInsertAfterWordClick
+            OnClick = cbInsertAfterWordClick
+            OnKeyUp = cbInsertAfterWordKeyUp
+            Items.Strings = (
+              '(space U+0020)'
+              '(no word break)'
+              '(Tibetan tsheg U+0F0B)'
```

Comment:
Does this string array also need zero-width space?

Reply:
I should remove those -- they are filled at runtime.